src/python/pants/backend/helm/target_types.py

```diff
@@ -183,5 +221,67 @@ class HelmUnitTestTestsGeneratorTarget(TargetFilesGenerator):
     help = f"Generates a `{HelmUnitTestTestTarget.alias}` target per each file in the `{HelmUnitTestGeneratingSourcesField.alias}` field"
 
 
+# -----------------------------------------------------------------------------------------------
+# `helm_artifact` target
+# -----------------------------------------------------------------------------------------------
+
+
+class HelmArtifactRegistryField(StringField):
+    alias = "registry"
+    help = (
+        "Registry alias (prefixed by `@`) configured in `[helm.registries]` for the Helm artifact"
+    )
+
+
+class HelmArtifactRepositoryField(StringField):
+    alias = "repository"
+    help = "Either an alias (prefixed by `@`) to a classic Helm repository configured in `[helm.registries]` or a path inside an OCI registry"
```

Comment:
FWIW, we're now recommending using `softwrap` for long/multi-line strings: see https://github.com/pantsbuild/pants/blob/ee214872958a75d2e3950bf9a938c608872d7323/src/python/pants/backend/docker/subsystems/docker_options.py#L39-L60 for example.